index.js

```diff
@@ -107,16 +110,23 @@ async function getRepoFromArguments() {
 
   console.log('Pull requests deployed to staging since the last deployment to production are:\n');
 
-  for (const pr of newPRs) {
-    console.log(`- ${pr.user.login}: ${pr.title}`);
+  const uniqueUsers = new Set()
+
+  for (const i = 0; i < newPRs.length; ++i) {
+    const pr = newPRs[i]
+    uniqueUsers.add(pr.user.login)
+    const userName = chalk.blue(pr.user.login + ":")
+    const mergeDate = chalk.green(DateTime.fromISO(pr.merged_at).toLocaleString(DateTime.DATETIME_MED))
```

Comment:
I believe `pr.merged_at` is a JS date. So `DateTime.fromJSDate` should be more appropriate.

Reply:
it's a string, this call would fail if it was a date

![Screen Shot 2022-03-09 at 5 14 35 PM](https://user-images.githubusercontent.com/66044732/157545585-82e0d418-5b44-4a84-ac32-eabdc7cbf418.png)